mailgun/tests.py

```diff
@@ -918,6 +918,215 @@ def test_multi_mailing_list_recipients_with_always_mail_staff_true(self, mock_ml
         )
         ml.send_mail.assert_has_calls([send_mail_call, send_mail_call])
 
+    @patch('mailgun.route_handlers.SuperSender.objects.filter')
+    @patch('mailgun.route_handlers.CourseInstance.objects.get_primary_course_by_canvas_course_id')
+    @patch('mailgun.route_handlers.MailingList.objects.get_or_create_or_delete_mailing_list_by_address')
+    def test_sender_receives_copy_when_always_mail_staff_false(self, mock_ml_get, mock_ci_get, mock_ss_filter):
+        """
+        TLT-2960
+        Verifies that when the  course_settings is False, the Sender gets a copy
```

Comment:
nitpick: it would be helpful to specify the exact setting (in this case, `always_mail_staff`)